src/poggit/module/build/RepoListBuildPage.php

```diff
@@ -104,9 +104,9 @@ protected function displayRepos(array $repos) {
             $opened = "false";
             if(count($repo->projects) === 1) $opened = "true";
             ?>
-            <div class="toggle" data-name="<?= $repo->full_name ?> (<?= count($repo->projects) ?>)"
+            <div class="repotoggle" data-name="<?= $repo->full_name ?> (<?= count($repo->projects) ?>)"
                  data-opened="<?= $opened ?>">
-                <h2><a href="<?= $home ?>ci/<?= $repo->full_name ?>">Projects</a> in
+                <h2><a href="<?= $home ?>ci/<?= $repo->full_name ?>"></a>
```

Comment:
If you look at a fully populated page it's so obvious that they are projects there's no need to repeat the word 20 times, not as a heading at least, or as the first word. the link is maybe useful which might be why I didn't remove it completely... but the text itself offers no information. The same goes for "Repo Enabled? Change", it's not necessary and just clutters things up